language/Dutch/strings.po

```diff
@@ -1287,7 +1287,7 @@ msgstr "Dipswitch"
 
 #: src/frontend/mame/ui/custui.cpp:460
 msgid "Unavailable color"
-msgstr "Onbeschikbare kleur"
+msgstr "Onbeschikbaar kleur"
```

Comment:
was correct, kleur is een "de" woord

Reply:
Same as previous. It's the color of items marked as unavailable in the UI. In the menu it makes more sense..